lib/containers/container_images.pm

```diff
@@ -54,4 +54,65 @@ sub build_with_zypper_docker {
 sub build_with_sle2docker {
 }
 
+# Testing openSUSE based images
+sub test_opensuse_based_image {
+    my $image = $_[0];
+    my ($name, $tag) = split(/:/, $image);
+    $tag //= 'latest';
+
+    my $runtime = $_[1] //= "docker";
+
+    my $distri  = $_[2] //= get_required_var("DISTRI");
+    my $version = $_[3] //= get_required_var("VERSION");
+
+    # It is the right version
+    if ($distri eq 'sle') {
```

Comment:
Why not `is_sle`?

Reply:
Because later we'd like to be able to test openSUSE images also on SLE and vice versa.

Reply:
SLE images on openSUSE won't have any repositories as the host isn't registered against SCC with a SLE product. So some tests would not work.

Reply:
They would, as it's mentioned here: https://github.com/SUSE/container-suseconnect#building-images-on-non-sle-distributions